build.gradle

```diff
@@ -19,15 +25,30 @@ repositories {
 }
 
 apply plugin: 'com.github.johnrengelman.shadow'
-apply plugin: 'java'
```

Comment:
is it not needed anymore?

Reply:
we don't need this plugin. it's a kotlin project.